spec/uncruft/deprecate_attribute_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+
+RSpec.describe Uncruft::DeprecateAttribute do
+  let(:my_name) { "Jess" }
+
+  subject { klass.new }
+
+  describe '#deprecate_attribute' do
+    context 'when aliased_attribute is present' do
+      let(:klass) do
+        Class.new do
+          include Uncruft::DeprecateAttribute
+
+          attr_accessor :first_name
+
+          deprecate_attribute(:first_name,
+                              aliased_attribute: :legal_first_name,
+                              message: "Please stop using this attribute, use this other attribute instead!")
+        end
+      end
+
+      it 'applies deprecation warning when setting and getting deprecated attribute' do
+        expect(ActiveSupport::Deprecation).to receive(:warn)
```

Comment:
If we're going to test the writer and reader together in a single spec, we probably want to be [more specific](https://relishapp.com/rspec/rspec-mocks/docs/setting-constraints/receive-counts)

```ruby
expect(...).to receive(...).exactly(n).times
```

Or we could break this up into two separate specs and set the reader spec up with `instance_variable_set`.

Reply:
I can just break them into seperate specs! Good call!